onnxruntime/python/tools/transformers/models/bart/utils/chain_enc_dec_with_beamsearch.py

```diff
@@ -0,0 +1,136 @@
+# -------------------------------------------------------------------------
```

Comment:
Is this used for testing or for the ONNX conversion process? Why it is needed?

Reply:
Done remove it.